pykern/pkunit.py

```diff
@@ -184,88 +186,86 @@ def file_eq(expect_path, *args, **kwargs):
     """
     import pykern.pkjson
     import pykern.pkconst
-    from pykern.pkdebug import pkdp
-
-    def _convert_xlsx(expect_path, actual_path):
-        if expect_path.ext == '.csv' and not actual_path.check(file=True):
-            _xlsx_to_csv(pkcollections.PKDict(
-                expect=expect_path,
-                actual=actual_path
-            ))
-
-    def _set_expect_and_actual(a, actual, actual_path, expect_path, is_jinja):
-        if expect_path.ext == '.json' and not actual_path.exists():
-            expect = pkio.read_text(expect_path)
-            if a:
-                import pykern.pkjson
-                pkio.mkdir_parent_only(actual_path)
-                actual = pykern.pkjson.dump_pretty(actual, filename=actual_path)
-        else:
-            if is_jinja:
-                import pykern.pkjinja
 
-                expect = pykern.pkjinja.render_file(expect_path, kwargs['j2_ctx'], strict_undefined=True)
-            else:
-                expect = pkio.read_text(expect_path)
-            if a:
-                pkio.write_text(actual_path, actual)
-        return pkcollections.PKDict(actual=actual, expect=expect)
-
-    def _set_expect_and_actual_paths(actual_path, expect_path):
-        if not isinstance(expect_path, pykern.pkconst.PY_PATH_LOCAL_TYPE):
-            expect_path = data_dir().join(expect_path)
-        is_jinja = expect_path.ext == '.jinja'
-        b = expect_path.purebasename if is_jinja else expect_path.relto(data_dir())
-        if actual_path is None:
-            actual_path = b
-        if not isinstance(actual_path, pykern.pkconst.PY_PATH_LOCAL_TYPE):
-            actual_path = work_dir().join(actual_path)
-        return pkcollections.PKDict(
-            actual_path=actual_path,
-            expect_path=expect_path,
-            is_jinja=is_jinja
+    class _FileEq:
```

Comment:
Move the class definition to just above the private methods.

Reply:
The class definition is at the top of ```file_eq``` definition. Not sure what you mean.

Reply:
Put the class definition just above here: https://github.com/radiasoft/pykern/blob/59c7a09381785caad7aeb4a15813de30c537d9d5/pykern/pkunit.py#L514

file_eq should just refer to it, not nest the class definition. There's no value in nesting a class definition.